tests/main/dbus-activation-session/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: D-Bus session services support activation via systemd user session
+
+systems:
+    # Ubuntu Core does not contain the D-Bus user session systemd units
+    - -ubuntu-core-*
```

Comment:
I think you could expand that to support core20 now.

Reply:
In PR #8943, I've made this test conditional on `tests.session has-session-systemd-and-dbus`.